go/src/meguca/server/init.go

```diff
@@ -114,7 +119,13 @@ func Start() error {
 	default:
 		return err
 	}
+
 	setConfigDefaults(&conf)
+	err = checkGeoLite()
+
+	if err != nil {
+		return err
```

Comment:
I really don't like that any of these have the potential to halt the server. Best emit a warning instead.

Reply:
I thought that might be an issue too, but wanted to get your input on it.
I'll do that, then.